README.md

```diff
@@ -36,22 +36,22 @@ Links to forums and info:
 * TR forum link: http://www.tombraiderforums.com/showthread.php?t=197508
 * Discord channel: https://discord.gg/d8mQgdc
 
-### Why it's developed? ###
+### Why create a new engine? ###
```

Comment:
I've made slight changes to the wording throughout, in order to make it flow a little better. Please, do not take any offence to this :slightly_smiling_face:

Reply:
Excellent

Reply:
Just a nitpick. It was probably unambiguous a year ago when OT was the only game in town. Now it could be misleading, because someone not familiar with the time-line of alternative TR engines could wonder what's wrong with them.